### ISSUE TITLE
A11y_AzureDataStudioAAE_Sql Instance 1_Connection Strings_DeveloperTool: No name property is provided for the edit box "ADO.NET" 

### ISSUE THREAD

**Issue by krrish2021** [microsoft/azuredatastudio#15750]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_AAE_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#DevTools;#WCAG4.1.2;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio -AAE
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till Azure Arc Controller tree and tab to the node present under it
4. Tab to child node "Sqlinstance1" and open manage and tab to "Connection Strings" page
5. Tab to "ADO.NET" edit box hit "ctrl + shift + i" and check if it has name provided.

### Actual:
No name property is provided for the edit box "ADO.NET". Similar issue is observed for other edit boxes present on the given screen.

similar issue is observed in the scenario given below:

1. PG1_Properties
2. PG1_Connection Strings

### Expected:
Name property to the edit box should be assigned according to it's label which is "ADO.NET"

### User Impact:
Screen Reader users will not get the proper information about the edit box.

**Comment by krrish2021** [comment]
Closing as not repro
![Screenshot (178)](https://user-images.githubusercontent.com/85863543/150954854-c5268e2a-43e9-4b31-89f0-58fd7c1ab99f.png)

**Comment by Harika1702** [comment]
GitHub Tags: #Closed;
